syserol/figures/figure9.py

```diff
@@ -0,0 +1,140 @@
+from syserol.model import (
+    Function_Prediction_10FoldCV,
+    SVM_2class_predictions,
+)  
+from sklearn.metrics import r2_score
+from syserol.figures.common import subplotLabel, getSetup
+from syserol.tensor import perform_CMTF
+
+
+def makeFigure():
+    """ Show Similarity in Prediction of Alter Model and Our Model"""
+    # Gather Function Prediction Accuracies
+    arry = Function_Prediction_10FoldCV(6)  # Our Function Predictions in an array
+    _, mapped = importFunction()
+    accuracies = np.zeros(12)
+    for ii, func in enumerate(mapped):
+        _, _, acc = function_elastic_net(func)  # Alter Function Predictions
+        accuracies[ii] = acc  # store accuracies
+    for i in np.arange(6):
+        x = arry[0:181, i]
+        y = arry[0:181, i + 6]
+        idx = np.isfinite(x)
+        accuracies[i + 6] = np.sqrt(
+            r2_score(x[idx], y[idx])
+        )  # Calculate our accuracies & store
+
+    # Create DataFrame
+    model = np.array(
+        [
+            "Alter Model",
+            "Alter Model",
+            "Alter Model",
+            "Alter Model",
+            "Alter Model",
+            "Alter Model",
+            "Our Model",
+            "Our Model",
+            "Our Model",
+            "Our Model",
+            "Our Model",
+            "Our Model",
+        ]
+    )
+    function = np.array(
+        [
+            "ADCD",
+            "ADCC",
+            "ADNP",
+            "CD107a",
+            "IFNy",
+            "MIP1b",
+            "ADCD",
+            "ADCC",
+            "ADNP",
+            "CD107a",
+            "IFNy",
+            "MIP1b",
+        ]
+    )
+    data = {"Accuracy": accuracies, "Model": model, "Function": function}
+    functions = pd.DataFrame(data)
+
+    # Gather Class Prediction Accuracies
+    accuracyCvP, accuracyVvN, _, _ = two_way_classifications()  # Alter accuracies
+    # Run our model
+    cube, glyCube = createCube()
+    tensorFac, matrixFac, R2XX = perform_CMTF(cube, glyCube, 6)
+    subjects_matrix = tensorFac[1][0]
+    cp_accuracy, nv_accuracy = SVM_2class_predictions(subjects_matrix)  # Our accuracies
+
+    # Create DataFrame
+    baselineNV = 0.5083  # datasetEV3/Fc.array/class.nv/lambda.min/score_details.txt "No information rate"
+    baselineCP = 0.5304  # datasetEV3/Fc.array/class.cp/lambda.min/score_details.txt "No information rate"
+    avg = np.mean([baselineNV, baselineCP])
+    accuracies = np.array(
+        [accuracyCvP, cp_accuracy, baselineCP, accuracyVvN, nv_accuracy, baselineNV]
+    )
+    category = np.array(
+        ["Progression", "Progression", "Progression", "Viremia", "Viremia", "Viremia"]
+    )
+    model = np.array(
+        ["Alter Model", "Our Model", "Baseline", "Alter Model", "Our Model", "Baseline"]
+    )
+    data = {"Accuracies": accuracies, "Class": category, "Model": model}
+    classes = pd.DataFrame(data)
+
+    # PLOT DataFrames
+    ax, f = getSetup((10, 7), (1, 2))
```

Comment:
Realistic size.

Reply:
Sorry, can you explain what exactly you mean by a realistic size? I was a bit confused about this, as in what units size is in and what realistic means? What is the frame of reference for sizes?

Reply:
The figure size (10, 7) is in inches. The width should probably never be over 8, and height above ~6-7.